### DISCUSSION TITLE
TeamTalk

### DISCUSSION THREAD

**Discussion by xyzs0000** [BearWare/TeamTalk5#1758] [Q&A]
### Description

Hello, we are an app group from China. We have been using TeamTalk for many years. I would like to ask a few questions. The first question: How to apply for the public release of our server in the qtTeamTalkg official server, what conditions must be met, The second one is that the public release in the Fei official server is also operated according to his specifications. Why is it not working? I can’t find it in the Fei official. After testing, as long as the server is in China, the release is not successful. This is why, other countries Yes, please reply, thank you

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [X] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
You can publish your server using the Qt TeamTalk client. Press F2 and locate the "Publish" button.
